account/src/main/java/com/caldevsupplychain/account/model/Company.java

```diff
@@ -1,13 +1,6 @@
 package com.caldevsupplychain.account.model;
 
-import javax.persistence.CascadeType;
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.OneToOne;
-import javax.persistence.Table;
+import javax.persistence.*;
```

Comment:
![screen shot 2017-11-17 at 1 36 42 pm](https://user-images.githubusercontent.com/31681641/32970103-6e3cd79a-cb9c-11e7-994d-9b2e7a894e3c.png)
Prefer import individual objects
https://stackoverflow.com/questions/10592951/in-java-when-should-i-import-whole-packages-with-instead-of-individual-object

Reply:
fix